File: content-solutions/graph-representation/graph-one-way-streets.java
class Solution {
    public int[][] solution(int n, int[][] roads) {
        int[] out = new int[n];
        int[] inn = new int[n];
        for (int[] road : roads) {
            out[road[0]]++;
            inn[road[1]]++;
        }
        int[][] answer = new int[n][2];
        for (int i = 0; i < n; i++) answer[i] = new int[] {out[i], inn[i]};
        return answer;
    }
}

File: content-solutions/graph-representation/graph-one-way-streets.js
function solution(n, roads) {
  const out = new Array(n).fill(0);
  const inn = new Array(n).fill(0);
  for (const [a, b] of roads) {
    out[a] += 1;
    inn[b] += 1;
  }
  return out.map((o, i) => [o, inn[i]]);
}
